src/dotnet-publish-iis/project.json

```diff
@@ -14,12 +14,21 @@
     "Microsoft.Extensions.Configuration.Json": "1.0.0-*",
     "Microsoft.DotNet.ProjectModel": "1.0.0-*",
     "Microsoft.DotNet.Cli.Utils": "1.0.0-*",
-    "NETStandard.Library": "1.5.0-*",
     "System.Diagnostics.Process": "4.1.0-*"
   },
 
   "frameworks": {
-    "dnxcore50": {
+    "netstandard1.5": {
```

Comment:
It is a tool but it is referencing the Microsoft.NETCore.App meta-package trying to be a shared .NET Core app so the correct TFM is netcoreapp1.0 for those.